src/components/Modals/ReleaseDetailsModal.js

```diff
@@ -74,15 +74,17 @@ class ReleaseDetailsModal extends React.Component {
                   </p>
 
                   <div className='release-selector-modal--components'>
-                    {_.sortBy(release.components, 'name').map((component) => {
-                      return (
-                        <ReleaseComponentLabel
-                          key={component.name}
-                          name={component.name}
-                          version={component.version}
-                        />
-                      );
-                    })}
+                    {_.sortBy(release.components, 'name').map(
+                      (component, i) => {
+                        return (
+                          <ReleaseComponentLabel
+                            key={`${component.name}-${release.version}-${i}`}
```

Comment:
Not ideal (talking about the index), but since I believe these aren't filtered, it shouldn't be a problem for now

Reply:
This is only required because aws release 9.2.3 contains [`coredns`](https://github.com/giantswarm/releases/blob/master/aws.yaml#L378-L379) [twice](https://github.com/giantswarm/releases/blob/master/aws.yaml#L408-L409) and we're putting everything together (`compoents` vs `apps` in the release). This would go away over time so I'm fine with reverting this to its original state.

Reply:
AWS release 9.2.4 will fix this inconsitency. I'll revert this later..